flytekit/annotated/base_task.py

```diff
@@ -33,6 +35,57 @@ def kwtypes(**kwargs) -> Dict[str, Type]:
     return d
 
 
+@dataclass
+class TaskMetadata(object):
+    """
+    Create Metadata to be associated with this Task
+
+    Args:
+      cache: Boolean that indicates if caching should be enabled
+      cache_version: Version string to be used for the cached value
+      interruptible: Boolean that indicates that this task is of for interruptions and can be scheduled on nodes
```

Comment:
"is of for interruptions" ?

Reply:
this is just copy pasta, but will fix it